evap/evaluation/models.py

```diff
@@ -266,9 +268,15 @@ def is_not_fully_reviewed(self):
         return self.open_textanswer_set.exists()
 
     @property
+    def actual_evaluation_end_datetime(self):
```

Comment:
i would want this to be called actual_vote_end_datetime to be consistent with vote_*_date, but what if we just called this vote_end_datetime?